runtime/base/base.Dockerfile

```diff
@@ -275,13 +275,13 @@ RUN set -xe; \
 #   - zlib
 # Needed by:
 #   - php
-ENV VERSION_XML2=2.9.14
+ENV VERSION_XML2=2.10.2
 ENV XML2_BUILD_DIR=${BUILD_DIR}/xml2
 
 RUN set -xe; \
     mkdir -p ${XML2_BUILD_DIR}; \
 # Download and upack the source code
-    curl -Ls https://download.gnome.org/sources/libxml2/2.9/libxml2-${VERSION_XML2}.tar.xz \
+    curl -Ls https://download.gnome.org/sources/libxml2/2.10/libxml2-${VERSION_XML2}.tar.xz \
```

Comment:
Can we use parameter expansion to avoid this mistake later?

```
➜  ~ VERSION_XML2=2.10.2
➜  ~ echo ${VERSION_XML2}   
2.10.2
➜  ~ echo ${VERSION_XML2%.*}
2.10
```

`%.*` will remove all characters from the end of the string until the dot.